ProjFS.Mac/Scripts/Build.sh

```diff
@@ -8,11 +8,11 @@ fi
 SCRIPTDIR=$(dirname ${BASH_SOURCE[0]})
 SRCDIR=$SCRIPTDIR/../..
 ROOTDIR=$SRCDIR/..
-PACKAGES=$ROOTDIR/packages 
+PACKAGES=$ROOTDIR/packages
 
 PROJFS=$SRCDIR/ProjFS.Mac
 
-xcodebuild -sdk macosx10.13 -configuration $CONFIGURATION -workspace $PROJFS/PrjFS.xcworkspace build -scheme PrjFS -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native || exit 1
```

Comment:
Why did you remove the sdk flag here? On my machine, this was required for it to build properly (back when I wrote the script anyway, not sure about today)

Reply:
Xcode 10 is out now, which removes the 10.13 SDK and replaces it with 10.14. So we can update all of our machines to 10 and update these strings to 10.14, or remove them altogether. I think we only needed the SDK flag in the old days when we were using functions that weren't in the 10.12 SDK. 

We also may have misconfigured our Xcode project in the past to not have a default SDK setup, but that is no longer the case :)